src/include/miopen/tensor_reorder_util.hpp

```diff
@@ -235,9 +234,8 @@ MakeTensorReorderAttributes(const ExecutionContext& ctx_,
     case 5:
         return std::make_unique<BatchedTransposeSolution_3012>(
             ctx_, data_type_, dim_0_, dim_1_, dim_2_, dim_3_);
-    default: MIOPEN_THROW("Unsupported reorder sequence"); break;
+    default: MIOPEN_THROW("Unsupported reorder sequence"); return nullptr;
```

Comment:
[Informative] Does this file really pass into static checks? `return nullptr` is still unreachable code.

Reply:
Updated in new commit.

Reply:
Resolved!